File: C#/BubbleSort.cs
public class BubbleSort {
    public int[] SortArray(int[] nums) {
        int temp = 0;
        for(int i=0;i<nums.Length-1;i++)
        {
            for(int j=1;j<nums.Length-i;j++)
            {
                if(nums[j-1]>nums[j])
                {
                    temp = nums[j-1];
                    nums[j-1] = nums[j];
                    nums[j] = temp;
                }

            }
        }
        return nums;
    }
}

File: Golang/BubbleSort.go
// You can edit this code!
// Click here and start typing.
package main

import "fmt"

func main() {
	var arr = [5]int{14, 655, 631, 2, 34}
	fmt.Println(arr)

	Bubblesort(&arr)

	fmt.Println(arr)

}

func Bubblesort(arr *[5]int) {

	temp := 0
	for i := 1; i < len(*arr); i++ {
		for j := 0; j < len(*arr)-i; j++ {

			if (*arr)[j] > (*arr)[j+1] {
				temp = (*arr)[j]
				(*arr)[j] = (*arr)[j+1]
				(*arr)[j+1] = temp
			}

		}

	}

}
